.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+# Stale config for https://github.com/marketplace/stale
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - security
+  - blocked
+
+# Label to use when marking a pull request as stale
+staleLabel: stale
+
+pulls:
+  # Number of days of inactivity before a pull request becomes stale
+  daysUntilStale: 7
+
+  # Number of days of inactivity before a stale pull request is closed
+  daysUntilClose: 7
+
+  # Comment to post when marking a pull request as stale. Set to `false` to disable
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs.
+
+  # Comment to post when closing a stale pull request. Set to `false` to disable
+  closeComment: >
+    This stale pull request has been automatically closed.
+    Thank you for your contributions.
+
+issues:
+  # Number of days of inactivity before a issue becomes stale
+  daysUntilStale: 365
```

Comment:
I am unsure about this one. Can we add a label instead of actually closing?
I could imagine feature requests wanting to live for longer.

Reply:
re-opening if still relevant remains an option

I think removing the stale label counts as activity, which would un-stale it both ways

Reply:
For what it's worth, it _really_ annoys me when projects auto-close issues and I have to keep reopening them...  I'd prefer we didn't do that, as an issue filer, if we can avoid it.

Importantly, is the goal here to prune issues or to surface issues that might have fallen through the cracks?  If the latter, auto-closing is not a good way to do it.

Reply:
in my experience, issues that are older than a year generally need re-triage

in the early phases of a project (like, < 2 years) issues get filed that become irrelevant for a myriad of reasons.  in mature projects, an issue that takes more than a year to address is usually a "won't fix" or an odyssey.  we can put "odyssey" labels on the latter.

nothing about this is permanent

Reply:
re-triage I absolutely agree with.  And yes, we can re-evaluate as we go, true.

Reply:
Can we set it up, with minimal effort, so some set of people get notified when something is marked stale and can re-triage?